src/components/molecules/timeline-events/return.tsx

```diff
@@ -38,24 +38,12 @@ const Return: React.FC<ReturnRequestedProps> = ({ event, refetch }) => {
     })
   }
 
-  const { mutateAsync: receiveReturn } = useAdminReceiveReturn(event.id)
+  const args = buildReturn(event, handleCancel, openReceiveReturnMenu)
```

Comment:
suggestion(non-blocking): Rename to something more descriptive
```suggestion
  const eventContainerArgs = buildReturn(event, handleCancel, openReceiveReturnMenu)
```